Confus/Confus/Audio/PlayerAudioEmitter.cpp

```diff
@@ -3,19 +3,34 @@
 
 #include "PlayerAudioEmitter.h"
 #include "AudioManager.h"
+#include "../../ConfusShared/Player.h"
 
 namespace Confus
 {
     namespace Audio
     {
-        PlayerAudioEmitter::PlayerAudioEmitter(irr::scene::ISceneNode* a_AttachedPlayer, AudioManager* a_AudioManager)
-			: m_AttachedPlayer(a_AttachedPlayer),
-			m_HitSoundHeavy(a_AudioManager->createSound("SFX/Player/a_heavy_grunt.wav")),
+		PlayerAudioEmitter::PlayerAudioEmitter(ConfusShared::Player* a_AttachedPlayer, AudioManager* a_AudioManager)
+			: m_HitSoundHeavy(a_AudioManager->createSound("SFX/Player/a_heavy_grunt.wav")),
 			m_HitSoundLight(a_AudioManager->createSound("SFX/Player/a_light_grunt.wav")),
 			m_HitSoundHeavyBackstab(a_AudioManager->createSound("SFX/Player/a_heavybackstab_grunt.wav")),
-			m_HitSoundLightBackstab(a_AudioManager->createSound("SFX/Player/a_lightbackstab_grunt.wav"))
+			m_HitSoundLightBackstab(a_AudioManager->createSound("SFX/Player/a_lightbackstab_grunt.wav")),
+			m_AttachedPlayer(a_AttachedPlayer)
         {
 			createAudioSources(a_AudioManager);
+			m_AttachedPlayer->getHealthInstance()->DamageEvent += [this](EHitIdentifier a_HitIdentifier) -> void
+			{
+				//playHitSound(a_HitIdentifier);
+			};
+
+			m_AttachedPlayer->OnHeavyAttack += [this]() -> void
+			{
+				//playHeavyAttack();
+			};
+
+			m_AttachedPlayer->OnLightAttack += [this]() -> void
+			{
+				//playLightAttack();
```

Comment:
Yes they are, thanks for pointing it out since I forgot. I do think it should be fixed separately, though. Guess I'll have to install audacity myself